content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
+
+          {% block pricing_opensource %}
+            <div class="column">
+              <div class="ui raised teal segment">
+                <h2 class="ui small monospace center aligned header">
+                  Community
+                </h2>
+
+                <p class="ui center aligned segment">
+                  Always <b>free</b> for <b>open source and community</b> projects.
+                </p>
+
+                <div class="ui list relaxed divided">
+                  {{ pricing.plan_feature("Concurrent builds", "2 concurrent builds", "fa-rectangle-vertical-history fa-swap-opacity") }}
+                  {{ pricing.plan_feature("Public projects", "Host public documentation", "fa-globe") }}
+                  {{ pricing.plan_feature("Public repositories", "Clone public repositories", "fa-globe") }}
+                  {{ pricing.plan_feature("Custom domain", "Use your own domain", "fa-at") }}
+                  {{ pricing.plan_feature("Ad supported", "Hosted documentation includes ads", "fa-ad") }}
                 </div>
               </div>
-
             </div>
-          </div>
-        </div>
-        <!-- end card -->
-
-        <!-- Card -->
-        <div class="ui card">
-
-          <div class="content">
-
-            <div class="ui label ribbon orange big">Ad-free</div>
-
-            <div class="ui header huge"><small><i>from</i></small> $5</div>
-            <div class="meta">
-              <span class="category">one-time donation</span>
-              <span class="right floated"><i>limited time</i></span>
+          {% endblock pricing_opensource %}
+
+          {% call pricing.plan(
+            name="Gold membership",
+            price="$5",
+            color="teal",
+          ) %}
+            <div class="ui list relaxed divided">
+              {{ pricing.plan_feature("Public versions", "Publicly accessible projects only", "fa-shield-keyhole") }}
+              {{ pricing.plan_feature("Ad free", "Turn off ads on your projects", "fa-ad", color="light grey") }}
             </div>
-
-            <div class="description">
-
-              <div class="ui list relaxed divided">
-                <div class="item">
-                  <div class="content">
-                    <div class="header"><i>Upgrade to an Ad-free user, for a limited period of time, with all the features from the open source plan</i></div>
-                  </div>
-                </div>
+          {% endcall %}
+
+          {% block pricing_choosing %}
+            <div class="column">
+              <div class="ui padded inverted teal segment">
+                <h3 class="ui small header">
+                  <i class="fad fa-circle-question icon"></i>
+                  <span class="content">
+                    Which platform?
+                  </span>
+                </h3>
+
+                <p>
+                  Read the Docs Community is intended for <b>open source</b> and <b>free</b> software,
```

Comment:
I assume we're using free as in beer here? We should probably be more explicit. We used `open source and community` elsewhere on the page, perhaps do the same here?

Reply:
Good point, yeah I'll update to match